coq/server/registrants/omnifunc.py

```diff
@@ -164,7 +164,6 @@ async def _resolve(nvim: Nvim, stack: Stack, user_data: UserData) -> UserData:
                 else:
                     return replace(
                         user_data,
-                        primary_edit=comp.primary_edit,
```

Comment:
@zeertzjq 

lmao, found out whats causing the `textlab` issue.

I think what the issue was, it was actually because `nvim` doesnt send `document/*change` during insertion mode, the LSPs view of the document is out of sync.

Reply:
so when you do a resolve for like lsp imports, it just gives you the old results.

wild